runtime/near-vm-runner/RUNTIMES.md

```diff
@@ -0,0 +1,214 @@
+This document describes the considerations that go into picking a Wasm runtime as well as the
+conclusions we made after evaluating a number of VM backends and frontends.
+
+# Criteria
+
+There are a number criteria which have been used to evaluate a Wasm runtime for use in a NEAR
+validator. Some of the criteria are already listed in the [FAQ] document and here we take a more
+thorough look. Listed roughly in the order of importance:
+
+* security – how well does the runtime deal with untrusted input;
+* correctness – does the runtime do what it claims to do;
+* reliability – how much confidence is there in the implementation of runtime;
+* platform support – does the runtime support targets that we want to target;
+* performance – how quickly can the runtime execute the requested operations;
```

Comment:
This is interesting. What is reason for putting platform support above performance?

Reply:
I reasoned about this in https://github.com/near/nearcore/pull/5881#discussion_r771518710. In short, I think our requirements to support `x86_64-linux` bring this above performance. But given that the question comes up a second time I'm wondering if either the description is lacking or this really should be less important compared to performance.

Reply:
Guess we need to define "requested operations".

Reply:
I want to avoid defining specifics in this list of criteria as the requirements section that comes after is better suited for elaboration in this aspect.

Re-reading the section on the performance requirements below I think it does list the kinds of operations we tend to ask a runtime to execute, but I do agree that the wording does not necessarily make the connection obvious.